<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab4ChartPlot</class>
 <widget class="QWidget" name="Tab4ChartPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2,1,3,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pLExPosition"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pLEyPosition"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBAddButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBClearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
